draft-bbs-signatures.md

```diff
@@ -117,6 +117,9 @@ commitment
 nonce
 : A cryptographic nonce
 
+presentation_message (pm)
+: A message generated and protected by an spk.
```

Comment:
I wouldn't say the PM is protected by the SPK, but it serves to tie the SPK to a specific context, ie. the current session between the prover and verifier.